test/functional/kaui/admin_allowed_users_controller_test.rb

```diff
@@ -18,6 +18,10 @@ class AdminAllowedUsersControllerTest < Kaui::FunctionalTestHelper
       post :create, parameters
       assert_equal 'User was successfully configured', flash[:notice]
       assert_response 302
+
+      # validate redirect path
+      id = get_allowed_id @response.body
+      assert_equal "/kaui/admin_allowed_users/#{id}", URI(@response.get_header('Location')).path
```

Comment:
Nit: would be nice to extract `URI(@response.get_header('Location')).path` into a test helper method (as we will need it in more and more places).